src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk

```diff
@@ -26,30 +28,48 @@ Before you create the new detailed guide or add the form pattern to an existing
+More than one type of form can be hosted on the same page. You can still only use the green button once on the page (check the <a href="/patterns-hmrc-govuk-team/green-button/" class="govuk-link">green button pattern</a> for more information).</li>
 </p>
 
-<h3 class="govuk-heading-m" id="Detailed examples">Detailed guidance examples</h3>
+<h3 class="govuk-heading-m" id="Detailed-examples">Forms in a detailed guide examples</h3>
+<ul class="govuk-list govuk-list--bullet">
+
+<li><a href="https://www.gov.uk/guidance/claim-a-tax-refund-if-youve-stopped-work-and-flexibly-accessed-all-of-your-pension-p50z" class="govuk-link">Claim a tax refund if you've stopped work and flexibly accessed all of your pension (P50Z)</a> —  online service with a green button, an interactive guidance (print and post) form, and a PDF (print and post) form for digitally excluded users</li>
+
+<li><a href="https://www.gov.uk/guidance/register-a-non-resident-company-who-disposed-of-uk-property-or-land-for-corporation-tax" class="govuk-link">Apply as an individual to receive rental income without UK tax deducted</a> — online service and LC print and post form</li>
+
+
+<li><a href="https://www.gov.uk/guidance/tell-hmrc-about-a-car-provided-to-an-employee-for-private-use" class="govuk-link">Tell HMRC about a car provided to an employee for private use</a> — example of online service and LC print and post form</li>
+
+<li><a href="https://www.gov.uk/guidance/research-and-development-tax-relief-advance-assurance" class="govuk-link">Apply for advance assurance on a Research and Development (R&D) tax relief claim</a> — online service, LC form, Interactive PDF that you complete online</li>
+</ul>
+
+
+<h3 class="govuk-heading-m" id="Examples-of-guides-with-one-form">Examples of guides with one form</h3>
 <ul class="govuk-list govuk-list--bullet">
-  <li><a href="https://www.gov.uk/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a> - an example of an online service with a green button and LC form to register by print and post</li>
-  <li><a href="https://www.gov.uk/guidance/claim-a-tax-refund-if-youve-stopped-work-and-flexibly-accessed-all-of-your-pension-p50z" class="govuk-link">Claim a tax refund if you've stopped work and flexibly accessed all of your pension (P50Z)</a> —  example of a service with a green button and both an interactive guidance (print and post) form and a PDF (print and post) form for digitally excluded users</li>
-  </ul>
+<li><a href="https://www.gov.uk/guidance/register-a-non-resident-company-who-disposed-of-uk-property-or-land-for-corporation-tax" class="govuk-link"> 
+Register a non-resident company who disposed of UK property or land for Corporation Tax</a> — G-form</li>
+<li><a href="https://www.gov.uk/guidance/work-out-pay-from-an-umbrella-company" class="govuk-link"> 
```

Comment:
Line returns shouldn't be inside HREFs

Reply:
@Jon-Rowe-HMRC Whitespace in HTML is largely ignored so shouldn't be a problem here but it is arguably easier to read the code when on the same line.

https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace